spec/fixtures/exports/general_needs_log.xml

```diff
@@ -54,7 +54,7 @@
     <waityear>1</waityear>
     <postcode_full>NW1 5TY</postcode_full>
     <reasonpref>1</reasonpref>
-    <cbl>1</cbl>
+    <cbl>2</cbl>
```

Comment:
cal is a checkbox in real form, so it should only be 1 or 0. Is this not using the real form?

Reply:
i see. i wasn't aware. it'll be because i changed the factory which has global implications. let me change that back and take a look

Reply:
there seems to be explicit code to set to `2`. see https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/blob/main/app/services/exports/lettings_log_export_service.rb#L186

i'm guessing this is what the consumer wants?

Reply:
Oh, okay, you're right. That's odd. It should only really be true or false (set to 1 if true and 0 or nothing if false). I'd expect it to be set to 0 if it's 0 in the import 🤔

Reply:
Don't listen to me, the checkboxes are 0-1 in our domain, but looking at the data dictionary it seems like they need to be exported as 1-2. This looks fine